root/lib/systemd/system/titus-atlasd.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Atlas for container %i
+ConditionPathIsDirectory=/var/lib/titus-inits/%i/ns
+
+# If the service restarts more than 10 times in 30 seconds, let it die
+StartLimitIntervalSec=30
+StartLimitBurst=10
+
+[Service]
+EnvironmentFile=/var/lib/titus-environments/%i.env
+# Run as root (UID 0, GID 0) and with CAP_DAC_OVERRIDE so that containers with a `USER` instruction work
+ExecStart=/usr/bin/runc --root /var/run/docker/runtime-${TITUS_OCI_RUNTIME}/moby exec --user 0:0 --cap CAP_DAC_OVERRIDE ${TITUS_CONTAINER_ID} /titus/atlas-titus-agent/start-atlas-titus-agent
```

Comment:
Is the command line the same for atlasd?

Reply:
It would be cool if we could start a convention of just

`/titus/atlastd/start`

`/titus/$sidecar/start`

Reply:
Can you elaborate, I don't understand.

Reply:
If some day we move to dynamic sidecars instead of encoding them here, having a convention for what the command is will help a bit.

For now in this review, I just wanted to confirm that `/titus/atlas-titus-agent/start-atlas-titus-agent` was the actual path (and not something with `atlasd` in it)

Reply:
Ah. It's the actual path.